R/reportPrices.R

```diff
@@ -316,8 +317,67 @@ reportPrices <- function(gdx, output=NULL, regionSubsetList=NULL,
       setNames(abs(esm2macro.m[,,name_trsp[1]]/(budget.m+1e-10)) * tdptwyr2dpgj , "Price|Energy Service|Transport LDV (US$2005/GJ)"))
   }
 
+  FE_data   <- reportFE(gdx, regionSubsetList = regionSubsetList, t = t)
+  Emi_data <- reportEmi(gdx, regionSubsetList = regionSubsetList, t = t)
+  FE_data_regi  <-  FE_data[getItems(pm_pvpRegi, dim = "all_regi"), , ] # get rid of GLO
+  Emi_data_regi <- Emi_data[getItems(pm_pvpRegi, dim = "all_regi"), , ]
+
+  # report GHG taxes, differentiated by sector
+  if (all(p21_CO2TaxSectorMarkup == 0)) { # then all are identical
```

Comment:
Please also check that the new parameter ``p21_CO2TaxSectorMarkup`` exists with ``is.null(p21_CO2TaxSectorMarkup)``. We usually do this for some backwards compatability such that reporting does not get broken if you have not pulled REMIND for some days. However, maybe we can drop this requirement if ``p21_CO2TaxSectorMarkup`` really exists by default in all new runs, but I think that would be up to RSE to decide more generally here. I usually still implement this conditionally to not break other people's reporting.

Reply:
If `p21_CO2TaxSectorMarkup` does not exist in the gdx file, then `readGDX` sets it to `NULL`. And `all(NULL == 0)` is `TRUE`, so your case is already covered.